docs/source/demo_callback.json

```diff
@@ -0,0 +1,90 @@
+{
+    "files": [
+        {
+            "primaryFilePath": "LNII/sbest/sbest-2023-0526-Rename/TEM-Tomo-2023-0526-Rename1/LNII/F13-TS005.czi",
```

Comment:
Perhaps we could have a more spacialopics like path?
"TEM-Tomo-2023-0526-Rename1" -> "SpacialOpimix_example-2023-0712"